price-feeder/oracle/oracle.go

```diff
@@ -277,8 +258,46 @@ func (o *Oracle) SetPrices(ctx context.Context, acceptList oracletypes.DenomList
 			return err
 		}
 
+		// warn the user of any missing prices
+		reportedPrices := make(map[string]struct{})
+		for _, providers := range providerPrices {
+			for base := range providers {
+				if _, ok := reportedPrices[base]; !ok {
+					reportedPrices[base] = struct{}{}
+				}
+			}
+		}
```

Comment:
Should we account for faulty providers here? e.g. 
```suggestion
		reportedPrices := make(map[string]struct{})
		for _, providers := range filteredProviderPrices {
			for base := range providers {
				if _, ok := reportedPrices[base]; !ok {
					reportedPrices[base] = struct{}{}
				}
			}
		}
```

wdyt @alexanderbez ?